foodsaving/conversations/api.py

```diff
@@ -190,6 +192,12 @@ def remove_reaction(self, request, pk, name):
         reaction.delete()
         return Response(status=status.HTTP_204_NO_CONTENT)
 
+    def perform_create(self, serializer):
+        message = serializer.save()
+        if message.conversation.target_type == ContentType.objects.get_for_model(Group):
```

Comment:
I lately used `isinstance(message.conversation.target, Group)`, but this seems fine as well. We could decide for one version to unify our styles :)

Reply:
Agree! Comparison using isinstance looks better. I've just committed updated version